manifests/init.pp

```diff
@@ -21,6 +21,10 @@
 #   See http://www.aptly.info/#configuration
 #   Default: {}
 #
+# [*create_config*]
+#   Whether to create a config file or not
+#   Default: true
```

Comment:
On a second thought, I'll actually replace this parameter with `single_root`, and create `/etc/aptly.conf.d` if it's set to `false`.